spec/system/bus_stops/editing_bus_stops_spec.rb

```diff
@@ -0,0 +1,102 @@
+
+require 'spec_helper'
+
+describe 'editing a bus stop as a user' do
+  let!(:user) { create :user }
+  let!(:edit_stop) { create :bus_stop }
+  before :each do
+    when_current_user_is user
+    visit edit_bus_stop_path(edit_stop.hastus_id)
+  end
+  context 'with no changes' do
+    before :each do
+      within 'table.edit-form' do
+        click_button 'Save stop'
+      end
+    end
+    it 'updates the stop' do
```

Comment:
Strictly speaking this doesn't do what it says on the tin. A message saying that the stop wasn't updated isn't proof positive that the stop was updated.

We could be huge liars.